ee-feature-pack/galleon-common/src/main/resources/layers/standalone/observability/layer-spec.xml

```diff
@@ -2,7 +2,6 @@
 <layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="observability">
     <dependencies>
         <layer name="microprofile-config" optional="true"/>
-        <layer name="microprofile-metrics" optional="true"/>
```

Comment:
We may have to redefine / override this in the 'wildfly' f-p as all of these layers are optional people might be excluding them using the 'microprofile-*' name.

If they are using 'wildfly-ee' then they have no choice; they have to change their exclude to 'metrics'.  But if they use 'wildfly' their existing exclude could work.

@jfdenise @yersan WDYT? Overriding a layer isn't nice either, and having a fixed definition for 'observability' is also a good thing.

Reply:
@bstansberry +1 on redefining the observability layer in wildfly FP to add back the optional dependency on the microprofile-metrics.

Reply:
I don't see the new "metrics" base layer in observability. User can't replace their exclusion to "metrics". I missed something there?

Reply:
@jfdenise https://github.com/wildfly/wildfly/blob/9c2367ec9fad676658cea070bf6e9e8af95270cf/ee-feature-pack/galleon-common/src/main/resources/layers/standalone/observability/layer-spec.xml#L4 or am I misunderstanding what you mean?

Reply:
@jmesnil , thank-you I should have looked at a commit, not the actual full PR. The 'metrics' is there, that answers my question.

Reply:
> WDYT? Overriding a layer isn't nice either, and having a fixed definition for 'observability' is also a good thing.

microprofile-config and microprofile-health subsystems will be also moved out from wildfly-ee. I'm not sure what observability will contain in wildfly-ee. It seems it will tend to be a complete different layer on wildfly-ee vs wildfly fp. Re-define it sounds although it seems users will see a different server configuration when using wildfly-ee and wildfly, so the same scripts won't work for the same layer. Am I wrong?

Reply:
In wildfly-ee observability will contain 'health' and 'metrics', the two most important aspects of observing a server. It won't have any form of tracing. Having MP Config was always more an implementation detail not a feature of 'observability'.

So, I do think the observability concept makes sense for wildfly-ee.

Yes, redefining the layer means what your configuration looks like will be different depending on what top-level FP you use. That does make it more unlikely someone can use the same script for both. Or the script becomes more complex.

I think if people were customizing they are going to have to adapt no matter what we do. For example the 'security-enabled' attribute on the mp-metrics-smallrye subsystem is no longer functional. And if you're using wildfly-ee the mp subsystems are just gone.

Reply:
My concern was oriented toward those cases thinking that since observability is going to have notable differences between the fp consumed we should have different names for such layer. I wanted to be sure it's assumable.

Then it means an user using the ee-wildfly fp that wants to incorporate MP specs moving to wildfly fp, would need to redefine / adapt any CLI scripts if he is using observability Galleon layer (or any layer that depends upon it, e.g cloud-profile and cloud-server)